packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsRichTextEditBox.tsx

```diff
@@ -92,7 +95,31 @@ export const ChatMessageComponentAsRichTextEditBox = (
     onCancelInlineImageUpload
   } = props;
 
-  const [textValue, setTextValue] = useState<string>(message.content || '');
+  const initialContent = useMemo(() => {
+    /* @conditional-compile-remove(rich-text-editor-image-upload) */
+    const content = message.content;
+    /* @conditional-compile-remove(rich-text-editor-image-upload) */
+    const document = new DOMParser().parseFromString(content ?? '', 'text/html');
+    /* @conditional-compile-remove(rich-text-editor-image-upload) */
+    document.querySelectorAll('.broken-image-wrapper').forEach((brokenImage) => {
+      const imageElement = document.createElement('img');
+      const attributes = brokenImage.attributes;
+      for (const attribute of attributes) {
```

Comment:
Why do we need to copy the attributes here?

Reply:
> Why do we need to copy the attributes here?

When John converted the original image to broken image, he copied all the attributes and now we need to copy it back to get the id, width and height etc.